packages/core/lib/inspect.js

```diff
@@ -0,0 +1 @@
+module.exports = require('q-i').print
```

Comment:
1. `@webpack-blocks/webpack` does not actually require or export `inspect()`.
2. Why not move the `inspect()` code to `@webpack-blocks/webpack` directly? It's not really core stuff, I'd say.

Reply:
I can move it to `webpack` — I don’t see any difference between webpack and core.

Reply:
I think `@webpack-blocks/webpack` feels more natural, since it's utility functionality, not core functionality.